tests/library/hit-target.spec.ts

```diff
@@ -464,3 +464,41 @@ it('should click in custom element', async ({ page }) => {
   await page.locator('input').click();
   expect(await page.evaluate('window.__clicked')).toBe(true);
 });
+
+for (const mode of ['same-context', 'different-context'] as const) {
+  it(`concurrent selectOption in ${mode} should work`, async ({ browser, browserName, isLinux }) => {
```

Comment:
Note #1: Was not sure in which file to put this test, since it needs to be a library test (not a page test).

Note #2: It passes in Firefox and Chromium.